src/assetchains.json

```diff
@@ -191,82 +174,196 @@
+    "addressindex": "1",
+    "spentindex": "1",
+    "addnode": [
+      "144.76.217.232"
+    ]
+  },
+  {
+    "ac_name": "KSB",
+    "ac_supply": "1000000000",
+    "ac_end": "1",
+    "ac_public": "1",
+    "addnode": [
+      "37.187.225.231"
+    ]
+  },
+  {
+    "ac_name": "OUR",
+    "ac_reward": "1478310502",
+    "ac_halving": "525600",
+    "ac_cc": "42",
+    "ac_supply": "100000000",
+    "ac_perc": "77700",
+    "ac_staked": "93",
+    "ac_pubkey": "02652a3f3e00b3a1875a918314f0bac838d6dd189a346fa623f5efe9541ac0b98c",
+    "ac_public": "1",
+    "addnode": [
+      "51.255.195.65",
+      "217.182.129.38",
+      "37.187.225.231"
+    ]
+  },
+  {
+    "ac_name": "ILN",
+    "ac_supply": "10000000000",
+    "ac_cc": "2",
+    "addnode": [
+      "51.75.122.83"
+    ]
+  },
+  {
+    "ac_name": "RICK",
+    "ac_supply": "90000000000",
+    "ac_reward": "100000000",
+    "ac_cc": "3",
+    "addnode": [
+      "138.201.136.145"
+    ]
+  },
+  {
+    "ac_name": "MORTY",
+    "ac_supply": "90000000000",
+    "ac_reward": "100000000",
+    "ac_cc": "3",
+    "addnode": [
+      "138.201.136.145"
+    ]
+  },
+  {
+    "ac_name": "VOTE2019",
+    "ac_supply": "123651638",
+    "ac_public": "1",
+    "addnode": [
+      "95.213.238.98"
+    ]
+  },
+  {
+    "ac_name": "KOIN",
+    "ac_supply": "125000000",
+    "addnode": [
+      "3.0.32.10"
+    ]
+  },
+  {
+    "ac_name": "ILN",
+    "ac_supply": "10000000000",
+    "ac_cc": "2",
+    "addnode": [
+      "51.75.122.83"
+    ]
+  },
+  {
+    "ac_name": "RICK",
+    "ac_supply": "90000000000",
+    "ac_reward": "100000000",
+    "ac_cc": "3",
+    "ac_staked": "10",
+    "addnode": [
+      "138.201.136.145",
+      "95.217.44.58"
+    ]
+  },
+  {
+    "ac_name": "MORTY",
+    "ac_supply": "90000000000",
+    "ac_reward": "100000000",
+    "ac_cc": "3",
+    "ac_staked": "10",
+    "addnode": [
+      "138.201.136.145",
+      "95.217.44.58"
+    ]
+  },
+  {
+    "ac_name": "KOIN",
+    "ac_supply": "125000000",
+    "addnode": [
+      "3.0.32.10"
+    ]
+  },
+  {
+    "ac_name": "ZEXO",
+    "ac_reward": "1478310502",
     "ac_halving": "525600",
+    "ac_cc": "42",
+    "ac_ccenable": "236",
+    "ac_supply": "100000000",
+    "ac_perc": "77700",
+    "ac_staked": "93",
+    "ac_pubkey": "02713bd85e054db923694b6b7a85306264edf4d6bd6d331814f2b40af444b3ebbc",
+    "ac_public": "1",
+    "addnode": [
+      "195.201.20.230",
+      "80.240.17.222"
+    ]
+  },
+  {
+    "ac_name": "K64",
+    "ac_reward": "0",
+    "ac_supply": "64000777",
+    "ac_staked": "10",
+    "addnode": [
+      "18.197.20.21"
+    ]
+  },
+  {
+    "ac_name": "HUSH3",
+    "ac_sapling": "1",
+    "ac_founders": "1",
+    "ac_reward": "0,1125000000,562500000",
+    "ac_end": "128,340000,5422111",
+    "ac_blocktime": "150",
+    "ac_supply": "6178674",
+    "ac_halving": "129,340000,840000",
     "ac_cc": "2",
-    "ac_pubkey": "039d4a50cc70d1184e462a22edb3b66385da97cc8059196f8305c184a3e21440af",
```

Comment:
This line cannot be removed. It is the founder's reward address and part of consensus.